iteration2_test_ex24.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import unittest
+from iteration2_exercise24 import(
+    strings_contain_only_letters,
+    equal_string_length,
+    strings_contain_same_letters,
+)
+
+class AnagramChecker(unittest.TestCase):
+    """Tests to see if two strings are anagrams"""
+
+    def test_only_letters(self):
+        """Test to see if both strings contain only letters"""
+
+        user_string_1 = 'note'
+        user_string_2 = 'tone'
+
+        strings_with_only_letters = strings_contain_only_letters(user_string_1, user_string_2)
+
+        self.assertTrue(strings_with_only_letters, True)
```

Comment:
I believe `assertTrue` only requires one argument that you're asserting is `True`:

```
self.assertTrue(strings_with_only_letters)
```

Reply:
This is repeated in your other functions.